java/java.lsp.server/vscode/package.json

```diff
@@ -401,6 +401,11 @@
 				"command": "foundProjects.deleteEntry",
 				"title": "Delete"
 			},
+      {
+                                "command": "micronaut.add.db.connection",
+                                "title": "Add Database Connection",
+ 				"category": "Micronaut"
```

Comment:
It'd be better if Micronaut extension were in `enterprise/micronaut` module, not here.